Source/WebKit/WebProcess/Inspector/ServiceWorkerDebuggableProxy.cpp

```diff
@@ -74,6 +74,18 @@ void ServiceWorkerDebuggableProxy::dispatchMessageFromRemote(String&& message)
         webProcessProxy->send(Messages::WebSWContextManagerConnection::DispatchMessageFromInspector(m_identifier, WTFMove(message)), 0);
 }
 
+#if ENABLE(REMOTE_INSPECTOR_SERVICE_WORKER_AUTO_INSPECTION) && ENABLE(REMOVE_XPC_AND_MACH_SANDBOX_EXTENSIONS_IN_WEBCONTENT)
+void ServiceWorkerDebuggableProxy::pauseWaitingForAutomaticInspection()
+{
+    m_isPausedWaitingForAutomaticInspection = true;
+}
+
+void ServiceWorkerDebuggableProxy::unpauseForInitializedInspector()
+{
+    m_isPausedWaitingForAutomaticInspection = false;
+}
+#endif
```

Comment:
Still need to call `RemoteInspector::singleton().setupCompleted()` to remove the debuggable from the `m_pausedAutomaticInspectionCandidates` and prevent the triggering of the 10 second timer set in `RemoteInspector::updateAutomaticInspectionCandidate`